File: tests/compatibility/v0.5.3/python/compatibility.py
"""Application wrapper retained with the UIMD 0.5.3 generated fixture."""

import sys

from uimd.runtime import UIApplication

from compatibility_ui import CompatibilityUI


class Compatibility(CompatibilityUI):
    pass


def main() -> int:
    application = UIApplication()
    application.open(Compatibility())
    return application.run()


if __name__ == "__main__":
    sys.exit(main())

File: tests/compatibility/v0.5.3/java/Compatibility.java
import uimd.GeneratedWindowRuntime;

public class Compatibility extends CompatibilityUI
{
    public static void main(String[] arguments)
    {
        int exitCode = GeneratedWindowRuntime.runGeneratedAppMain(() ->
        {
            Compatibility application = new Compatibility();
            return GeneratedWindowRuntime.runGeneratedWindow(
                application,
                application.runtimeOptions(),
                arguments);
        });
        System.exit(exitCode);
    }
}
